spec/controllers/api/v1/recent_jobs_controller_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::RecentJobsController, type: :controller do
+
+  describe "GET #index" do
+    let(:response_body) { json_response = JSON.parse(response.body) }
+
+    it "is successful" do
+      get :index, format: :json
+
+      expect(:success)
+    end
+
+    it 'returns jobs with correct attributes' do
+      create(:job)
+      get :index, format: :json
+
+      response_body['recent_jobs'].each do |job|
+        expect(job['title']).to be_instance_of(String)
+        expect(job['description']).to be_instance_of(String)
+        expect(job['url']).to be_instance_of(String)
+        expect(job['location']).to be_instance_of(String)
+        expect(job['posted_date']).to be_instance_of(String)
+        expect(job['remote']).to be false
+        expect(job['company']).to be_instance_of(Hash)
+      end
+    end
+
+    it 'returns the id and name of the company the job belongs to' do
+      create(:job)
+      get :index, format: :json
+      company = response_body['recent_jobs'].first['company']
+
+      expect(company['id']).to be_instance_of(Fixnum)
+      expect(company['name']).to be_instance_of(String)
+    end
+
+    it 'only returns jobs posted within the last 2 months' do
+      three_month_job = create(:job)
```

Comment:
I like the completeness of your tests here. Just FYI though, you can override attributes in Factory Girl, so it could look like create(:job, posted_date: 3.months.ago).